test/bump.sdkver.test.ts

```diff
@@ -61,13 +61,17 @@ beforeEach(() => {
   jest.spyOn(github, "isPullRequestEvent").mockReturnValue(false);
   jest.spyOn(github, "createTag").mockResolvedValue();
   jest.spyOn(github, "createRelease").mockResolvedValue();
+  jest.spyOn(github, "getCommitsBetweenRefs").mockResolvedValue([]);
 
```

Comment:
Adding a test case for the new behavior would be appreciated (but not mandatory for these changes to be accepted)